frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -202,7 +202,9 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
           }
           const secretId = secret?.items?.[0]?.metadata?.uid;
           if (secretId) {
-            redirect = `${config.redirectURL}?selectId=${secretId}&selectTab=Release+notes`;
+            redirect = `${config.redirectURL}?selectId=${secretId}&selectTab=${t(
+              'helm-plugin~Release+notes',
```

Comment:
I think it would also work if you use the same title here which already exists. The browsers will automatically add `+`  or `%20` to escape this.

```suggestion
              'helm-plugin~Release notes',
```